Opcodes/sfont.c

```diff
@@ -574,7 +591,7 @@ static int32_t SfPlay(CSOUND *csound, SFPLAY *p)
         }
         phs++; base++; sampinc++; endloop++; startloop++;
         left++; right++, mode++, end++; attack++; decay++; sustain++;
-        tinc++; env++; attr++; decr++;
+        /*release++;*/ tinc++; env++; attr++; decr++;
```

Comment:
Curious, why is the commented release++ added here and in other parts of commit?

Reply:
this is from earlier fixes in csound6, variables not used triggering warnings/errors.

Nothing to do with the android fix